src/Modules/Pages/Domain/RevisionBlock/RevisionBlockType.php

```diff
@@ -51,24 +56,16 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
         );
         $builder->add(
             'block',
-            EntityType::class,
+            EntityType::class,  // TODO should probably only fetch blocks for this locale?
```

Comment:
You are right, I pushed the fix for this on the fork6 branch, if you rebase you should get it and can remove this comment